src/content/tutorials/observability/data-ingestion/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Ingesting new data
+description: Guides explaining how to ingest observability data (metrics and logs) into the Observability Platform.
+weight: 20
+menu:
+  principal:
+    parent: tutorials-observability
+    identifier: tutorials-observability-data-ingestion
+last_review_date: 2024-07-11
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+user_questions:
+  - How can I ingest observability data?
```

Comment:
Also here good "easier" question without our specific vocabulary would be

*   How to add new data into the observability platform?

Reply:
```suggestion
  - How can I ingest observability data?
  - How to add new data into the observability platform?
```